2_adventures/001/kotp/04_automated_testing_example/spec/fizz_buzz_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'spec_helper'
+require 'fizz_buzz'
+
+describe FizzBuzz do
+  describe 'Single Numbers' do
+    it 'must return 1 for 1' do
```

Comment:
I think it's a matter of personal preference. Our brain is quite good at ignoring visual clutter that doesn't interest us. I agree that the `test_0000` prefix is superfluous and that the space helps scanning. I had a look at the `raindrops` Exercism task that is similar. Here's one of its tests:

```ruby
  def test_5
    assert_equal 'Plang', Raindrops.convert(5)
  end
```

So the method name only has the input value which in reverse mode for specs style would mean we'd only be interested in the output, i.e. `it 'returns "Fizz"'` and the test call in its block would give the parameter to be tested. Again, a matter of personal preference.